index.bs

```diff
@@ -136,4 +136,6 @@ navigator.audioSession.onstatechange = () => {
 
 # Acknowledgements # {#acknowledgements}
 
-TODO
+The Working Group acknowledges the following people for their invaluable contributions to this specification:
+
+* Becca Hughes
```

Comment:
```suggestion
* Becca Hughes
* Mounir Lamouri
* Zhiqiang Zhang
```

Reply:
This adds the other people I see in the WICG commit history.

Reply:
Sounds good